.circleci/config.yml

```diff
@@ -44,6 +55,22 @@ jobs:
           paths:
             - ~/test/vendor/bundle
 
+  build_js:
```

Comment:
misschien iets voor discussie, maar eigenlijk moeten we ervoor zorgen dat die translations in JS altijd gebuild worden, op de een of andere manier. Volgens mij kan het nu in theorie voorkomen dat een niet geupdate versie van translations.js toch naar productie gaat. Of niet?

Reply:
Ja dat kan. Maar ik weet niet of we dat in CircleCI kunnen doen eigenlijk. Want ten eerste hij deployt vanaf Heroku (heroku houdt een bepaalde branch in de gaten). Dus dan zou je iets moeten doen dat kijkt of er een commit binnenkomt en dan die translations.js genereren en dan bij een verschil opnieuw een commit doen dat is vet brak.

Het is nu wel zo dat translations.js automatisch gegenereerd wordt bij ieder webrequest. Het zit namelijk in de middelware, en het is zo ingesteld dat hij dat translations.js bestand up to date houdt. Het kan dus in theorie voorkomen dat je een translation aanpast in een nl.yml bestand ergens, en vervolgens niet je lokale dev server een willekeurige pagina laat renderen (waardoor translations.js geupdate zou worden) maar in plaats daarvan direct commit: en dan zou inderdaad de nieuwe vertaling nog niet in translations.js zitten.

Wat dacht je van het volgende: We voegen een check toe aan CircleCI, en het enige wat die check doet is die draait `rake i18n:js:export` (wat ook die translations.js genereert) gaat vervolgens kijken of public/translations.js is veranderd. Als dat het geval is dan faalt de test en dan gaat de CircleCI check dus niet goed.

Reply:
Fixed.